test/sagemaker_tests/tensorflow/tensorflow2_training/integration/sagemaker/test_smtrcomp.py

```diff
@@ -0,0 +1,399 @@
+                               image_uri=ecr_image,
+                               framework_version=framework_version,
+                               hyperparameters={
+                                    TrainingCompilerConfig.HP_ENABLE_COMPILER : True,
+                               },
+                               )
+        estimator.fit(mnist_dataset, job_name=unique_name_from_base('test-TF-trcomp'))
+        _assert_model_exported_to_s3(estimator)
+        captured = capsys.readouterr()
+        _assert_training_compiler_invoked(captured)
+
+
+    @pytest.mark.model('LeNet')
+    @pytest.mark.integration("s3 plugin")
+    def test_s3_plugin(self, sagemaker_session, ecr_image, framework_version, instance_type, instance_count, tmpdir, mnist_distributed_dataset, capsys):
+        script = os.path.join(resource_path, 'mnist', 'mnist_custom.py')
+        estimator = TensorFlow(entry_point=script,
+                               role='SageMakerRole',
+                               instance_type=instance_type,
+                               instance_count=instance_count,
+                               sagemaker_session=sagemaker_session,
+                               image_uri=ecr_image,
+                               framework_version=framework_version,
+                               hyperparameters={
+                                   # Saving a checkpoint after every 5 steps to hammer the S3 plugin
+                                   'save-checkpoint-steps': 10,
+                                   # Reducing throttling for checkpoint and model saving
+                                   'throttle-secs': 1,
+                                   # Without the patch training jobs would fail around 100th to
+                                   # 150th step
+                                   'max-steps': 200,
+                                   # Large batch size would result in a larger checkpoint file
+                                   'batch-size': 1024,
+                                   # This makes the training job exporting model during training.
+                                   # Stale model garbage collection will also be performed.
+                                   'export-model-during-training': True,
+                                    TrainingCompilerConfig.HP_ENABLE_COMPILER : True,
+                               },
+                               )
+        estimator.fit(mnist_distributed_dataset, job_name=unique_name_from_base('test-TF-trcomp-s3'))
+        _assert_checkpoints_exported_to_s3(estimator, 10)
+        captured = capsys.readouterr()
+        _assert_training_compiler_invoked(captured)
+
+
+    @pytest.mark.xfail(reason="SM Training Compiler team yet to implement this integration test")
+    @pytest.mark.integration("hpo")
+    def test_hyperparameter_tuner(self, sagemaker_session, ecr_image, framework_version, instance_type, instance_count, tmpdir, capsys):
```

Comment:
Lets add the `@pytest.mark.model` marker to this function as well

Reply:
Done